kratos/sources/variables.cpp

```diff
@@ -882,5 +886,9 @@ void KratosApplication::RegisterVariables() {
     KRATOS_REGISTER_3D_VARIABLE_WITH_COMPONENTS(INTEGRATION_COORDINATES)
 
     KRATOS_REGISTER_VARIABLE(TABLE_UTILITY)
+
+    // for DEM-FEM 2D
+    KRATOS_REGISTER_VARIABLE(IMPOSED_Z_STRAIN_VALUE)
```

Comment:
I would not move those to the core, to me are not general enough

Reply:
The thing is that we need those variables in a problem coupling the Structural Mechanics Application with the DEM Application, and there were no similar variables in Structural Mechanics that we could use. Thereby, in order to avoid repeating the same variable in 2 applications, we decided to move them to the core. 
In any case, if you have a better alternative, please let us know.

Reply:
I would actually repeat the variable, to me this would not be a problem. Since #1468 it is possible to define the same variable in multiple applications. 

As I said I think this is a quite specific variable that is not general enough for the Core
The Core already has many variables that were only put there because it was previously not possible to define the same variable in several applications

Since this is my opinion/suggestion and since it is not crucial, I will not block because of this.
I just would like to not bloat the Core even more

Reply:
Ok, then I'll try defining those variables in both applications ;)